modules/Timetable/moduleFunctions.php

```diff
@@ -1857,23 +1857,30 @@ function renderTTSpace($guid, $connection2, $gibbonSpaceID, $gibbonTTID, $title
                     $output .= "<th $today style='vertical-align: top; text-align: center; width: ";
 
                     $output .= (550 / $daysInWeek);
-                    $output .= "px'>";
+                    $output .= "px".$color."'>";
```

Comment:
Hmm, looks like another PR again. I'll see if I can merge that one first to resolve these issues. It is a good idea to start your branches fresh rather than from a previous PR branch.